pkg/azuredisk/azuredisk_v2.go

```diff
@@ -254,21 +268,25 @@ func (d *DriverV2) StartControllersAndDieOnExit(ctx context.Context) {
 	klogv1.SetOutputBySeverity("FATAL", klogWriter{})
 	log := klogr.New().WithName("AzDiskControllerManager").WithValues("namepace", d.objectNamespace).WithValues("partition", d.controllerPartition)
 
-	leaseDuration := time.Duration(d.controllerLeaseDurationInSec) * time.Second
-	renewDeadline := time.Duration(d.controllerLeaseRenewDeadlineInSec) * time.Second
-	retryPeriod := time.Duration(d.controllerLeaseRetryPeriodInSec) * time.Second
+	// leaseDuration := time.Duration(d.controllerLeaseDurationInSec) * time.Second
+	// renewDeadline := time.Duration(d.controllerLeaseRenewDeadlineInSec) * time.Second
+	// retryPeriod := time.Duration(d.controllerLeaseRetryPeriodInSec) * time.Second
```

Comment:
Why are these commented out? If not needed, then simply remove these lines.

Reply:
Leader election was not working with the initial configuration. No controller manager was trying to acquire the lock and they were all just hanging. So, I was testing with fields set to different values to see how the behavior differs and these are remnants from the testing. I am going to comment them back in later :)

Reply:
How much later is later? If you need to commit this PR and debug leader election later, please add a TODO comment.